tests/console/test_Log.py

```diff
@@ -145,8 +152,41 @@ def test_save_log_to_file_with_exception(tmp_path, now):
     with open(log_filepath, encoding="utf-8") as log:
         log_contents = log.read()
 
+    assert len(logger.stacktraces) == 1
+    assert log_contents.startswith("Date/Time:       2022-06-25 16:12:29")
+    assert TRACEBACK_HEADER in log_contents
+    assert log_contents.splitlines()[-1].startswith("ZeroDivisionError")
+
+
+def test_save_log_to_file_with_multiple_exceptions(tmp_path, now):
+    """Log file contains exception stacktrace when more than one is
+    captured."""
+    command = MagicMock()
+    command.base_path = Path(tmp_path)
+    command.command = "dev"
+    command.tools.os.environ = {}
+
+    logger = Log()
+    logger.save_log = True
+    for i in range(1, 5):
+        try:
+            1 / 0
+        except ZeroDivisionError:
+            logger.capture_stacktrace(f"Thread {i}")
+
+    logger.save_log_to_file(command=command)
+
+    log_filepath = tmp_path / logger.LOG_DIR / "briefcase.2022_06_25-16_12_29.dev.log"
+
+    log_filepath.exists()
```

Comment:
Should this be an `assert`? same in `test_save_log_to_file_with_exception()` it seems

Reply:
Yup - looks like a weird copy-and-paste error.